tests/basic/async_dr_test.go

```diff
@@ -67,7 +67,10 @@ var _ = Describe("{MigrateDeployment}", func() {
 	It("has to deploy app, create cluster pair, migrate app", func() {
 		Step("Deploy applications", func() {
 
-			SetSourceKubeConfig()
+			err := SetSourceKubeConfig()
+			if err != nil {
+				log.FailOnError(err, "Switching context to source cluster failed")
+			}
```

Comment:
Just do `log.FailOnError(err, "Switching context to source cluster failed")`
Checking for error is unnecessary because that is handled in `log.FailOnError`

Reply:
Oh, yeah...
✅